### PR TITLE
Make 'All pages' view label consistent with template and patterns

### PR DESCRIPTION
**Pull request by jameskoster**
## What?
Update the label on the data view for all pages.

## Why?
So that it's consistent with patterns and templates.

| Before | After |
| --- | --- |
| Screenshot 2024-02-14 at 11 42 13 | Screenshot 2024-02-14 at 11 39 54 |

## References
Screenshot 2024-02-14 at 11 43 22

Screenshot 2024-02-14 at 11 43 10

### CODE REVIEW

**Review by oandregal — APPROVED** [review]

**Comment by huzaifaalmesbah** [comment]
I check your changes. all looks good.

File: packages/edit-site/src/components/sidebar-dataviews/default-views.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ const DEFAULT_PAGE_BASE = {
 export const DEFAULT_VIEWS = {
 	page: [
 		{
-			title: _x( 'All', 'pages' ),
+			title: _x( 'All pages', 'pages' ),
```
Comment (t-hamano) [inline packages/edit-site/src/components/sidebar-dataviews/default-views.js]:
```suggestion
			title: __( 'All pages' ),
```

Context may no longer be necessary.

Note: If you apply this suggestion, you will also need to remove `_x` from the import statement to pass the lint check.

Comment (jameskoster) [reply packages/edit-site/src/components/sidebar-dataviews/default-views.js]: That's true, good suggestion :)

**Review by t-hamano — APPROVED** [review]
LGTM 🚀

**Comment by getdave** [comment]
I just cherry-picked this PR to the cherry-pick-beta-2 branch to get it included in the next release: 35f7226903

**Pull request merged**